src/main/flight/mixer.c

```diff
@@ -510,26 +527,33 @@ void mixTable(pidProfile_t *pidProfile)
     if (feature(FEATURE_3D)) {
         if (!ARMING_FLAG(ARMED)) throttlePrevious = rxConfig()->midrc; // When disarmed set to mid_rc. It always results in positive direction after arming.
 
-        if ((rcCommand[THROTTLE] <= (rxConfig()->midrc - flight3DConfig()->deadband3d_throttle))) { // Out of band handling
+        throttle3Dstate_e flight3DState = get3DState(throttlePrevious);
```

Comment:
This is just a step in between anyways. The final version might still be same in terms of performance